_inc/client/apps/index.jsx

```diff
@@ -0,0 +1,153 @@
+						</div>
+						<div className="jp-jetpack-landing__column">
+							<img src={ imagePath + '/apps/laptop-90deg-themes.png' } />
+						</div>
+					</div>
+				</div>
+
+				<div className="jp-jetpack-landing__img-text">
+					<div className="jp-jetpack-landing__column">
+						<img src={ imagePath + '/apps/manage2x' } />
+					</div>
+					<div className="jp-jetpack-landing__column">
+						<h2>{ __( 'Secure & Manage' ) }</h2>
+						<p>{ __( 'Most security flaws are found in outdated plugins. Use our Web and Desktop apps to turn on auto-updates or update plugins manually for all your websites in one convenient place.' ) }</p>
+						<Button href={ 'https://wordpress.com/plugins/' + window.Initial_State.rawUrl }	className="is-primary">
+							{ __( 'Manage on WordPress.com' ) }
+						</Button>
+					</div>
+				</div>
+
+				<div className="jp-jetpack-landing__img-text">
+					<div className="jp-jetpack-landing__column">
+						<h2>{ __( 'Focus on your Writing' ) }</h2>
+						<p>{ __( 'Our new editor is lightning fast, optimized for writers and eliminates distractions, giving you the ability to focus on your work.' ) }</p>
+						<Button href={ 'https://wordpress.com/post/' + window.Initial_State.rawUrl }	className="is-primary">
+							{ __( 'Write on WordPress.com' ) }
+						</Button>
+					</div>
+					<div className="jp-jetpack-landing__column">
+						<img src={ imagePath + '/apps/editor2x' } />
+					</div>
+				</div>
+
+				<div className="jp-jetpack-landing__img-text">
+					<div className="jp-jetpack-landing__column">
+						<img src={ imagePath + '/apps/stats2x' } />
+					</div>
+					<div className="jp-jetpack-landing__column">
+						<h2>{ __( 'Connect with your Visitors' ) }</h2>
+						<p>{ __( 'Monitor your visitors with advanced stats. Watch for trends, learn what content performs the best and understand your visitors from anywhere in the world.' ) }</p>
+						<Button href={ 'https://wordpress.com/stats/' + window.Initial_State.rawUrl }	className="is-primary">
+							{ __( 'View Stats on WordPress.com' ) }
+						</Button>
+					</div>
+				</div>
+
+				<div className="jp-jetpack-landing__img-text">
+					<div className="jp-jetpack-landing__column">
+						<h2>{ __( 'Connect with the Community' ) }</h2>
+						<p>{ __( 'The WordPress apps all have impressively fast and full featured readers so you can catch up with your favorite sites and join the conversation anywhere, any time.' ) }</p>
+						<Button href={ 'https://wordpress.com/reader' }	className="is-primary">
+							{ __( 'Browse WordPress.com' ) }
+						</Button>
+					</div>
+					<div className="jp-jetpack-landing__column">
+						<img src={ imagePath + '/apps/community2x' } />
+					</div>
+				</div>
+
+				<div className="jp-jetpack-landing__footer">
+					<h2 className="jp-jetpack-landing__title" title="Inspiration strikes any time, anywhere.">
```

Comment:
Should all `h2` elements have a title? If so, let's make them all translated as well. And if not, maybe we can remove it?